docs/dev/dev_setup.rst

```diff
@@ -44,6 +44,13 @@ with the following commands:
     $ mamba update mamba
     $ conda config --set channel_priority strict
 
+.. note::
+
+   Make sure that after you've installed ``conda`` or ``mamba`` that you can activate
+   and deactivate the ``base`` environment (which is created by default). Try creating
+   a simple new environment and activating it to make sure everything is working and
+   your shell has been correctly configured to use the new package manager.
+
```

Comment:
@denimalpaca would this have helped avoid the conda environment initialization problem you ran into?